app/views/admin/gdpr_delete_requests/index.html.erb

```diff
@@ -2,10 +2,20 @@
 <div id="gdpr-delete-requests-content" class="crayons-card overflow-admin-main-layout-padding p-4 m:p-0">
   <header class="flex flex-col l:flex-row justify-content-between l:items-center p-0 m:p-7 pb-4">
     <h1 class="crayons-title ml-3 l:ml-0">Members<span class="screen-reader-only"> (GDPR Delete Requests)</span></h1>
+    <%= form_with url: admin_gdpr_delete_requests_path, method: :get, local: true, class: "py-3" do |f| %>
```

Comment:
It looks like we have nested form fields, which is causing me some confusion and appears to be a violation of the [HTML form element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form):

> Permitted Content :: [Flow content](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#flow_content), but not containing `<form>` elements

We'll need to rework this.